_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -5703,6 +5703,16 @@ void MfxHwH264Encode::SetDefaults(
     if (extOpt2->RepeatPPS == MFX_CODINGOPTION_UNKNOWN)
         extOpt2->RepeatPPS = MFX_CODINGOPTION_ON;
 
+    if (extOpt2->MinQPI || extOpt2->MinQPP || extOpt2->MinQPB ||
+        extOpt2->MaxQPI || extOpt2->MaxQPP || extOpt2->MaxQPB)
+    {
+        if (!extOpt2->MaxQPI)
+           extOpt2->MaxQPI = 51;
```

Comment:
Will it work for >8bit samples?

Reply:
there is no >8bits support in h264 encoder